Library/Formula/git-vendor.rb

```diff
@@ -0,0 +1,18 @@
+class GitVendor < Formula
+  desc "command for managing git vendored dependencies"
+  homepage "https://brettlangdon.github.io/git-vendor"
+  head "https://github.com/brettlangdon/git-vendor.git"
+
+  stable do
```

Comment:
@DomT4 thanks, pushed an updated version that incorporates that info into the initial block